src/test/rgw/test_multi.py

```diff
@@ -965,11 +1034,17 @@ def init(parse_args):
     log_level = args.log_level
 
     global rgw_multi
+    master_zg_base_port = 8000
 
-    rgw_multi = RGWMulti(int(args.num_zones), int(args.gateways_per_zone))
-
+    rgw_multi = RGWMulti(1, int(args.num_zones), int(args.gateways_per_zone), master_zg_base_port, master_zg_base_port)
     rgw_multi.setup(not args.no_bootstrap, args.tenant)
 
+    for i in range(1, args.num_zonegroups):
+        rgw_multi = RGWMulti(i + 1, int(args.num_zones), int(args.gateways_per_zone), master_zg_base_port + 100 * i, master_zg_base_port)
+
+    for i in range(1, args.num_zonegroups):
+        rgw_multi.setup(not args.no_bootstrap, args.tenant)
+
```

Comment:
this breaks when there are more than 2 zonegroups - the second loop will repeatedly try to initialize the last zonegroup. need to combine the two for loops:
```python
    for i in range(1, args.num_zonegroups):
        rgw_multi = RGWMulti(i + 1, int(args.num_zones), int(args.gateways_per_zone), master_zg_base_port + 100 * i, master_zg_base_port)
        rgw_multi.setup(not args.no_bootstrap, args.tenant)
```

also, since `rgw_multi` isn't used elsewhere (and it no longer means what it used to), could you remove the `global rgw_multi`?